Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -276,7 +276,7 @@ class CreditCardBottomSheetViewController: UIViewController, UITableViewDelegate
             item: creditCard,
             isAccessibilityCategory: UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory)
         hostingCell.host(creditCardRow, parentController: self)
-
+        hostingCell.backgroundColor = themeManager.currentTheme.colors.layer5
```

Comment:
Can we make that `HostingTableViewCell` `ThemeApplicable` instead? This way it will get theme update whenever the theme is updated. With `cellForRowAt` it will only get theme updated if we reload the table view.